### DISCUSSION TITLE
I want to know the path to iOS Simulator SDK

### DISCUSSION THREAD

**Discussion by kanisuki** [airsdk/Adobe-Runtime-Support#3141] [Q&A]
I want to know the path to iOS Simulator SDK,
What to do? 
In case windows PC, it is easy, only click directories.

![quespackaging](https://github.com/airsdk/Adobe-Runtime-Support/assets/61077580/a0bb98ce-d178-4a1c-a638-20b4fc097b3e)

**Comment by ajwfrost** [comment]
The field there is for the Apple iPhone simulator; from the perspective of AIR/Animate, this is a 'target device' as we have to pack up an installer file and deploy it onto the simulator before the app can run. So, this is a mac-only thing, and you can find the path from a terminal window using:
```
 xcrun --show-sdk-path --sdk iphonesimulator
```
On Windows, you can't use the iPhone simulator so you will just need to use the "debug movie" option, which runs the app locally using ADL (AIR debug launcher) - but if you've got it set up with the "AIR for iOS" option then this does go into a 'mobile profile' mode, and you get a small tool alongside it to allow you to emulate some mobile-specific sensor inputs such as orientation etc.

thanks

**Reply by kanisuki** [reply]
Thank you.
I can see the path at MAC PC.
